src/Microsoft.ML.Transforms/CountFeatureSelectingTransformer.cs

```diff
@@ -40,28 +46,108 @@ public sealed class Arguments : TransformInputBase
+            /// </summary>
+            /// <param name="input">Name of the input column.</param>
+            /// <param name="output">Name of the column resulting from the transformation of <paramref name="input"/>. Null means <paramref name="input"/> is replaced. </param>
+            /// <param name="count">If the count of non-default values for a slot is greater than or equal to this threshold in the training data, the slot is preserved.</param>
+            public ColumnInfo(string input, string output = null, long count = Defaults.Count)
+            {
+                Input = input;
+                Contracts.CheckValue(Input, nameof(Input));
+                Output = output ?? input;
+                Contracts.CheckValue(Output, nameof(Output));
+                Count = count;
+            }
+        }
+
+        /// <include file='doc.xml' path='doc/members/member[@name="CountFeatureSelection"]' />
+        /// <param name="env">The environment to use.</param>
+        /// <param name="columns">Describes the parameters of the feature selection process for each column pair.</param>
+        public CountFeatureSelectingEstimator(IHostEnvironment env, params ColumnInfo[] columns)
+        {
+            Contracts.CheckValue(env, nameof(env));
+            _host = env.Register(RegistrationName);
+            _host.CheckUserArg(Utils.Size(columns) > 0, nameof(columns));
+
+            _columns = columns;
+        }
+
+        /// <include file='doc.xml' path='doc/members/member[@name="CountFeatureSelection"]' />
+        /// <param name="env">The environment to use.</param>
+        /// <param name="inputColumn">Name of the input column.</param>
+        /// <param name="outputColumn">Name of the column resulting from the transformation of <paramref name="inputColumn"/>. Null means <paramref name="inputColumn"/> is replaced. </param>
+        /// <param name="count">If the count of non-default values for a slot is greater than or equal to this threshold in the training data, the slot is preserved.</param>
+        public CountFeatureSelectingEstimator(IHostEnvironment env, string inputColumn, string outputColumn = null, long count = Defaults.Count)
+            : this(env, new ColumnInfo(inputColumn, outputColumn ?? inputColumn, count))
+        {
+        }
+
+        public SchemaShape GetOutputSchema(SchemaShape inputSchema)
+        {
+            _host.CheckValue(inputSchema, nameof(inputSchema));
+            var result = inputSchema.Columns.ToDictionary(x => x.Name);
+            foreach (var colPair in _columns)
+            {
+                if (!inputSchema.TryFindColumn(colPair.Input, out var col))
+                    throw _host.ExceptSchemaMismatch(nameof(inputSchema), "input", colPair.Input);
+                if (!CountFeatureSelectionUtils.IsValidColumnType(col.ItemType))
+                    throw _host.ExceptSchemaMismatch(nameof(inputSchema), "input", colPair.Input);
```

Comment:
>ExceptSchemaMismatch [](start = 32, length = 20)

fix the error message: now it will say that the column is missing, but needs to mention expected type